src/osp/Resource/load_tinygltf.cpp

```diff
@@ -136,7 +136,7 @@ static void load_gltf(TinyGltfImporter &rImporter, ResId res, std::string_view n
         }
 
         // Create and keep track of resource Id
-        ResId const imgRes = rResources.create(gc_image, pkg, format_name(rImporter.image2DName(i), i));
+        ResId const imgRes = rResources.create(gc_image, pkg, SharedString::create(format_name(rImporter.image2DName(i), i)));
```

Comment:
No way to implicitly create shared strings from other standard strings? This can also be moved to the format_name lambda instead.

Reply:
The reason to avoid doing that is to avoid surprise allocations.

I made the sharedstring class require the factory function for things other than copy/move construction to specificly make it hard to get one by accident.